services/portfolios_ca/src/services/targets.ts

```diff
@@ -9,45 +9,91 @@ export default () => {
      *
      * @returns {Object} Target basic information and values of portfolios
      */
-    getPortfoliosByTarget: async () => ({
-      target_id: 3,
-      target_name: 'Servicios Ecosistémicos',
-      target_national: 4521,
-      target_units_short: 'TC',
-      target_units: 'Toneladas de carbono',
-      portfolios_data: [
+    getPortfoliosByTarget: async (targetId: number) => {
+      const data = [
         {
-          id: 1,
-          name: 'NATURE MAP · WCMC',
-          short_name: 'WCMC',
-          value: 100,
+          target_id: 2,
+          target_name: 'Ecosistemas',
```

Comment:
please change these props to target_id: targetId (the param) and add that same param to the name, so in the front we see 8 instead of 2 groups

Reply:
Done, now the service return the id and name for all the targets